Misc/NEWS.d/next/Core and Builtins/2018-04-26-22-40-08.bpo-33358.VRMsp2.rst

```diff
@@ -0,0 +1,2 @@
+Fix `test_pre_initialization_sys_options` when building with --enable-
```

Comment:
There is a better section for your fix: Tests.

If you write a NEWS entry, mention at least the name of the test (file): test_embed. Example:

```
Fix ``test_embed.test_pre_initialization_sys_options()``
```